whd/dispatcher.py

```diff
@@ -139,6 +145,89 @@ def _pipeline_definition_changed(self, push_event):
             return True
         return False
 
+    def determine_affected_pipelines(self, push_event) -> Pipeline:
```

Comment:
nit: typehint should by typing.Generator[Pipeline], should it not?
nit2: I would not necessarily include the yielding into the docstr (this is already documented in the return-typehint..) - not that important, though